doc/nrf/releases/release-notes-latest.rst

```diff
@@ -93,13 +96,13 @@ The following list summarizes the most important changes inherited from upstream
   * Deprecated ``BT_LE_SCAN_FILTER_WHITELIST``.
     Use :cpp:enumerator:`BT_LE_SCAN_OPT_FILTER_WHITELIST <bt_gap::BT_LE_SCAN_OPT_FILTER_WHITELIST>` instead.
   * Deprecated ``bt_le_scan_param::filter_dup``.
-    Use ``bt_le_scan_param::options`` instead.
+    Use :cpp:member:`bt_le_scan_param::options` instead.
   * Deprecated ``bt_conn_create_le()``.
     Use :cpp:func:`bt_conn_le_create` instead.
   * Deprecated ``bt_conn_create_auto_le()``.
-     Use :cpp:func:`bt_conn_le_create_auto` instead.
+    Use :cpp:func:`bt_conn_le_create_auto` instead.
   * Deprecated ``bt_conn_create_slave_le()``.
-    Use :cpp:func:`bt_le_adv_start` instead, with ``bt_le_adv_param::peer`` set to the remote peer's address.
+    Use :cpp:func:`bt_le_adv_start` instead, with :cpp:member:`bt_le_adv_param::peer` set to the remote peer's address.
```

Comment:
Ditto here, although now I see that various things are being documented in the C++ domain which should really be in the C domain.

Reply:
Same here - this is the only way to get the links working.